### DISCUSSION TITLE
Builds getting stuck

### DISCUSSION THREAD

**Discussion by rgdcastro** [vercel/community#4683] [Help] [answered]
### Summary

Has anyone else experiencing problems with the builds getting stuck? It was working fine before and build time is only around 2-3 mins. But now, it is reaching the limit of 45 minutes.

I am using Nuxt.

![image](https://github.com/vercel/community/assets/36554808/f0ab7ef1-17be-4346-9c65-18ea03285e82)

### Steps to Reproduce

Initiate a build.

**Comment by amyegan** [comment]
I wasn't able to reproduce this issue with my test projects. Do you have this problem with any of your other projects?

**Reply by rgdcastro** [reply]
I only have this one project in our Vercel account. This project simultaneously deploys to Cloudflare Pages though and the project builds there without any issues.

**Reply by rgdcastro** [reply, marked as answer]
We can close this now. Builds suddenly works fine again after 3+ hours.

**Comment by MTGPROD** [comment]
I have the same issue right now, but idk if I have to wait or if it's due to a thing that I misconfigured
